projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/components/no-linked-cases/no-linked-cases.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { LinkedCasesService } from '../../services/linked-cases.service';
+import { NoLinkedCasesComponent } from './no-linked-cases.component';
+
+describe('NoLinkedCasesComponent', () => {
+  let component: NoLinkedCasesComponent;
+  let fixture: ComponentFixture<NoLinkedCasesComponent>;
+  let router: any;
+  let linkedCasesService: any;
+  const caseId = '1682374819203471';
+
+  router = {
+    navigate: jasmine.createSpy('navigate'),
+    url: ''
+  };
+  router.navigate.and.returnValue({then: f => f()});
+
+  linkedCasesService = {
+    caseId: '1682374819203471',
+    getAllLinkedCaseInformation() {}
+  };
+
+  beforeEach(async(() => {
```

Comment:
@johnbenjamin-hmcts - change to `waitForAsync()`

Reply:
@Daniel-Lam I have updated the code now.